packages/agent-toolkit/src/shared/parameters.ts

```diff
@@ -0,0 +1,172 @@
+import { z } from "zod";
```

Comment:
lovely module!

Reply:
I didn't double check your zod defs here against any official source of truth... do we _have_ an official source of truth for these from which we can derive the zod schemas? Like json schema docs?